src/components/views/elements/ReplyThread.js

```diff
@@ -127,6 +127,8 @@ export default class ReplyThread extends React.Component {
             // for a timeline with that event, but once it is loaded we can use findEventById to look up the ev map
             await this.context.matrixClient.getEventTimeline(room.getUnfilteredTimelineSet(), eventId);
         } catch (e) {
+            // if it fails, capture the error and dump it because it'll make findEventById return false, which
+            // captures a superset of errors.
```

Comment:
This explanation wording is slightly confusing. It suggests that the error will cause `findEventById` to return a certain value but we `return null` immediately afterwards instead of falling through to `room.findEventById` below.

I suggest explaining why `null` is returned and why errors are treated the same way as missing events are (iiuc).